### DISCUSSION TITLE
How to determine gamma value (k) in CAFE5

### DISCUSSION THREAD

**Discussion by LeoVincenzi** [hahnlab/CAFE5#184] [Q&A] [answered]
Dear author,
I am running CAFE on an analysis of 4 genomes of the same genre. I have performed firstly a run of Orthofinder and now I am giving as input the following table.
```
Desc    HOG     Lalb_protein    Lang_protein    Lcos_protein    Ldig_protein
n/a     N0.HOG0000002   1       38      0       0
n/a     N0.HOG0000005   0       29      1       0
n/a     N0.HOG0000008   0       1       1       1
n/a     N0.HOG0000009   0       19      1       0
n/a     N0.HOG0000010   0       2       1       0
n/a     N0.HOG0000011   0       1       1       0
n/a     N0.HOG0000012   0       45      1       0
n/a     N0.HOG0000014   0       53      1       0
n/a     N0.HOG0000015   0       1       0       1
```

I tried the analysis with k=1,2,3,5,7 and observed similar results, but I would try to understand the logic behind using one value or another. If you could explain better this point, it would be great.

Thanks, 
Leo

**Comment by benfulton** [comment, marked as answer]
It shouldn't matter very much which you use. CAFE is trying to estimate the gamma curve based on the number of points you tell it to try, so if you tell it to use 7 points it might select the points below on the curve:
![image](https://github.com/user-attachments/assets/b31f102a-4674-4fff-ab76-b0d7b62cd30b)

If you tell it to use fewer points, say 3, it might use points 1,4,7 on the graph only, and that is likely to give you a slightly less accurate answer. But it probably won't make a very big difference.
